src/modules/account/selectors/core-stats.js

```diff
@@ -1,6 +1,5 @@
 import { createBigNumber } from "utils/create-big-number";
 import { createSelector } from "reselect";
-import store from "src/store";
```

Comment:
What are the changes in this file about?

Reply:
Ok these changes are fine. It seems they were just removed because they were not being used and Webstorm probably told me and I probably just removed them when I deleted that export.